docs/intro.rst

```diff
@@ -1,20 +1,29 @@
 
-The Data Activated 流 (Liu) Graph Engine (|daliuge|) is a workflow graph execution framework, 
-specifically designed to support very large scale processing graphs for the reduction of 
-interferometric radio astronomy data sets.
-|daliuge| aims to provide a distributed data management platform and a
-scalable pipeline execution environment to support continuous, soft real-time,
-data-intensive processing for producing radio astronomy data products.
+The |daliuge| system can and has been used on a huge range of different size systems. It can easily be deployed on a person's laptop, across multiple personal machines as well as small and large compute clusters. That flexibility comes with some difficulty in describing how the system is intended to be used, since that is obviously dependent on the way it is deployed. This guide mainly describes the basic usage of the system as it would appear when deployed on a single machine or a small cluster.
 
-|daliuge| originated from a prototyping activity as part of the SKA SDP Consortium called Data Flow Management System (DFMS).
+DALiuGE Ecosystem
+------------------
 
-The development of |daliuge| is largely based on radio astronomy processing requirements.
-However, |daliuge| has adopted a generic, data-driven framework architecture potentially applicable to
-many other data-intensive applications.
+A key part of the DALiuGE ethos is to reduce the amount of time scientists spend transitioning existing pipelines towards a new, distributed pipeline. This is achieved by: 
+
+* Visualising the logical relationship between different tasks in the pipeline (a **Logical Graph**);
+* Providing drop-in support for a range of existing application 'formats', including Docker containers and shell applications;
+* Removing the need to learn a new domain specifical language or scripting framework - existing code can be referenced directly in the graph without the need to produce intermediate file storage. 
+
+Depending on the existing use case and software being used in the pipeline, it is entirely possible that `no additional code` needs to be written in order to create a distributed pipeline that can be deployed across an entire cluster!
+
+In order to achieve this, DALiuGE is developed as part of an ecosystem of tools that each support each stage of pipeline development: 
+
+* DALiuGE Palette Generator Tool: This tool decomposes an existing Python module or library into DALiuGE Components, which are the basis of the visual logical pipeline;
+* Editor for the Astronomical Graph Language Environment (EAGLE): EAGLE is the graphical environment in which a user uses components to draw the Logical Graph, which represents the high-level interactions of the different parts of the workflow.  
+* DALiuGE: The workflow execution framework that manages the deployment and runtime demands of the pipeline. This 'translates' the Logical Graph into a complete workflow based on the runtime parameters provided in the Logical Graph. 
```

Comment:
and runtime demands of the workflow. This 'translates' the Logical Graph into an executable workflow based on runtime parameters and parallelization and flow constructs specified in the Logical Graph.